src/map.rs

```diff
@@ -13,438 +13,434 @@ pub enum MatcherType {
     CUSTOM,
 }
 
-pub fn setup(v: &mut Vec<(MatcherType, String, String, Matcher)>) {
-    // TODO
-    // Replace all of this with macro?
-
-    // Order: Application, Image, Video, Audio, Font, Document, Archive
```

Comment:
Can we leave the comment on the order please. Perhaps just above where we invoke the `matcher_map!` macro. The order is somewhat relevant in how we evaluate the matchers.